src/mbgl/layout/symbol_layout.cpp

```diff
@@ -90,10 +92,10 @@ SymbolLayout::SymbolLayout(std::string bucketName_,
                 u8string = platform::lowercase(u8string);
             }
 
-            ft.text = util::utf8_to_utf32::convert(u8string);
+            ft.text = bidi.bidiTransform( util::utf8_to_utf16::convert( u8string ) );
```

Comment:
I’m curious about how visual mirroring at this stage interacts with line wrapping. Does this PR address the concern raised in https://github.com/mapbox/mapbox-gl-native/pull/6057#issuecomment-240545901?